lib/install/management/store_files.go

```diff
@@ -53,15 +53,37 @@ func (d *Dispatcher) DeleteStores(vchVM *vm.VirtualMachine, conf *metadata.Virtu
 	var emptyImages bool
 	var emptyVolumes bool
 	log.Infof("Removing images")
-	if emptyImages, err = d.deleteImages(ds, p); err != nil {
-		errs = append(errs, err.Error())
+	for _, imageDir := range conf.ImageStores {
+		imageDSes, err := d.session.Finder.DatastoreList(d.ctx, imageDir.Host)
```

Comment:
>  another one that uses the same regexes to go from ds://host/path format to url.URL because that function doesn't exist currently)
https://github.com/vmware/vic/blob/master/pkg/vsphere/datastore/datastore.go#L331